cql3/statements/create_view_statement.cc

```diff
@@ -189,7 +189,7 @@ std::pair<view_ptr, cql3::cql_warnings_vec> create_view_statement::prepare_view(
         }, selectable);
 
         auto* def = get_column_definition(*schema, *identifier);
-        if (!def) {
+        if (!def || def->is_internal()) {
```

Comment:
Please correct me if I'm wrong, but here you only fix explicit SELECT of specific internal column names into a view.
But what about a view with `SELECT *` - it too should exclude the internal columns, and I don't know where is the logic to do that. See https://github.com/scylladb/scylladb/issues/21154 about how a view with `SELECT *` is implemented differently (include_all_columns()) than a SELECT of specific columns.